rudder-reports/debian/copyright

```diff
@@ -1,24 +1,15 @@
-This package was debianized by root <root@rudderVM.normation.com> on
-Mon, 17 Jan 2011 08:19:10 -0500.
+This package was debianized by:
 
-It was downloaded from <url://example.com>
+    Jonathan Clarke <Jonathan Clarke <jonathan.clarke@normation.com>> on Tue, 06 Oct 2009 01:20:26 +0200
```

Comment:
Perhaps there is a reason for the will of change. @amousset: why do you changed it?